### DISCUSSION TITLE
How to write a "ringfile" like ringbuffer?

### DISCUSSION THREAD

**Discussion by Fodesu** [gabime/spdlog#2927] [Q&A]
Hello everyone, I have a requirement: to print encrypted logs in a ring file similar to a ring buffer. How can I implement this?

**Comment by tt4g** [comment]
There is no such feature sink in spdlog, but you can create your own custom sink.
Wiki: https://github.com/gabime/spdlog/wiki/4.-Sinks#implementing-your-own-sink

Refer to ring_buffer_sink.
Replace buffer operations with file seek and write operations in your implementation.
